bookwyrm/views/list.py

```diff
@@ -200,6 +210,63 @@ def post(self, request, list_id):
         return redirect(book_list.local_path)
 
 
+class EmbedList(View):
+    """embeded book list page"""
+
+    def get(self, request, list_id, list_key):
+        """display a book list"""
+        book_list = get_object_or_404(models.List, id=list_id)
+
+        embed_key = str(book_list.embed_key.hex)
+
+        if list_key != embed_key:
+            raise Http404()
+
+        query = request.GET.get("q")
```

Comment:
is this used for anything? normally I think it would be searching to add books to a list, which you wouldn't do in this context. I think you can omit it

Reply:
Ooh yes you're right, I thought it was part of the List view